src/containers/location/providers/ProviderDetailsContainer.js

```diff
@@ -2,532 +2,50 @@
-  @media only screen and (min-height: ${HEIGHTS[2]}px) {
-    min-height: 350px;
-  }
-
-  @media only screen and (min-height: ${HEIGHTS[3]}px) {
-    min-height: 460px;
-  }
-
-  @media only screen and (min-height: ${HEIGHTS[4]}px) {
-    min-height: 630px;
-  }
-`;
-
-const StyledContentWrapper = styled(ContentWrapper)`
-  background-color: white;
-  position: relative;
-  padding: 0 ${PADDING}px ${PADDING}px ${PADDING}px !important;
-`;
-
-const Row = styled.div`
-  align-items: flex-start;
-  display: flex;
-  flex-direction: row;
-  font-size: 14px;
-  justify-content: space-between;
-  line-height: 19px;
-  margin: 8px 0;
-
-  div {
-    color: ${NEUTRAL.N700};
-    max-width: 65%;
-  }
-
-  a {
-    color: ${PURPLE.P300};
-    text-decoration: underline;
-    max-width: 65%;
-  }
-`;
-
-const DataRows = styled.div`
-  display: flex;
-  flex-direction: column;
-  ${(props) => (props.maxWidth ? css` max-width: ${props.maxWidth} !important; ` : '')}
-
-  span {
-    color: ${NEUTRAL.N600};
-    text-align: right;
-  }
-
-  a {
-    min-width: fit-content;
-    text-align: right;
-    ${(props) => (props.alignEnd ? css` align-self: flex-end; ` : '')}
-  }
-`;
-
-const TitleRow = styled.div`
-  align-items: center;
-  color: ${NEUTRAL.N600};
-  display: flex;
-  flex-direction: row;
-  justify-content: space-between;
-  padding: 20px 0;
-  width: 100%;
-
-  span {
-    color: ${NEUTRAL.N700};
-    font-size: 14px;
-    font-style: normal;
-    font-weight: 600;
-    line-height: 17px;
-
-    :last-child {
-      font-weight: normal;
-    }
-  }
-`;
-
-const DateRow = styled.article`
-  display: flex;
-  flex-direction: row;
-  justify-content: space-between;
-
-  span {
-    color: ${NEUTRAL.N600};
-  }
-
-  span:first-child {
-    margin-right: 10px;
-    text-align: left;
-  }
-
-  span:last-child {
-    text-align: right;
-  }
-`;
-
-const Line = styled.div`
-  background-color: ${NEUTRAL.N100};
-  height: 1px;
-  margin: ${(props) => props.paddingTop || 0}px -${PADDING}px 0 -${PADDING}px;
-`;
-
-const InfoText = styled.div`
-  color: ${NEUTRAL.N600};
-  font-size: 14px;
-  font-style: normal;
-  font-weight: normal;
-  line-height: 19px;
-`;
-
-type Props = {
-  hospital :Map;
-  provider :Map;
-  getText :(translation :Translation) => string;
-  rrs :Map;
+import { Line, StyledContentOuterWrapper, StyledContentWrapper } from '../../styled';
+
+const { LOCATIONS } = STATE;
+const { SELECTED_PROVIDER } = PROVIDERS;
+
+const ProviderDetailsContainer = () => {
+  const provider = useSelector((store) => store.getIn([LOCATIONS, SELECTED_PROVIDER]));
+
+  if (!provider) {
+    return null;
+  }
+
+  const sections = [
+    <FamilyHomeLocationSection />,
+    <VacancySection />,
+    <CapacitySection />,
+    <ContactSection />,
+    <HealthAndSafetySection />,
+    <ResourceAndReferralSection />
+  ].filter((s) => s).map((s, idx) => (
+    /* eslint-disable-next-line */
+    <Fragment key={idx}>
+      {s}
+      <Line />
+    </Fragment>
+  ));
```

Comment:
This is funky. It looks like the only reason you need the `.filter()` is to prevent `<Line />` from rendering when a component returns `null`, of which only one could possibly return `null` (I think it's only 1).

I think it'd be cleaner and more broadly reusable to create a styled wrapper that applies the `Line` effect automatically, something like...

```
const MyFancyStyledWrapper = styled.div`
  > section {
    border-bottom: 1px solid ${NEUTRAL.N100};
    padding: ...
    
    &:last-child {
      border-bottom: 0;
      padding: ...
    }
  }
`;
```

The LUK `CardSegment` component does the same thing.